pkgs/tools/text/fanficfare/default.nix

```diff
@@ -1,19 +1,29 @@
-{ stdenv, fetchurl, python27Packages }:
+{ stdenv, fetchFromGitHub, python2Packages }:
 
-python27Packages.buildPythonApplication rec {
-  version = "3.1.1";
-  name = "fanficfare-${version}";
-  nameprefix = "";
+python2Packages.buildPythonApplication rec {
+  pname = "FanFicFare";
+  version = "3.6.0";
 
-  src = fetchurl {
-    url = "https://github.com/JimmXinu/FanFicFare/archive/v${version}.tar.gz";
-    sha256 = "1wklii24vbvq2vi5pqgp3z4lazcplh2i7r2w4d8lkm6pzbw0s8px";
+  src = python2Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "1ir3m8wknr8shdbmbpiaw73mdpa7mvidkl6pbs9ca23mgwivxa84";
   };
+  # src = fetchFromGitHub {
+  #   owner = "JimmXinu";
+  #   repo = pname;
+  #   rev = "v${version}";
+  #   sha256 = "08c4dphm9hsm4cvg3pd03vbnmz6a5apl6hvn1w5kqpzbwi33i8j2";
+  # };
```

Comment:
Why?

Reply:
This is still WIP, and the variant obtained from PIP contains one less tests.

Reply:
Okay, and why not the `fetchFromGitHub` variant?

Reply:
It fails too.